frontend/lib/src/components/widgets/Checkbox/Checkbox.test.tsx

```diff
@@ -63,30 +65,10 @@ describe("Checkbox widget", () => {
     )
   })
 
-  it("has correct className and style", () => {
-    const props = getProps()
-    const wrapper = mount(<Checkbox {...props} />)
-
-    const wrappedDiv = wrapper.find("div").first()
-
-    const { className, style } = wrappedDiv.props()
-    // @ts-expect-error
-    const splittedClassName = className.split(" ")
-
-    expect(splittedClassName).toContain("row-widget")
-    expect(splittedClassName).toContain("stCheckbox")
-
-    // @ts-expect-error
-    expect(style.width).toBe(getProps().width)
-  })
-
```

Comment:
Think it's worth adding back for consistency and because we check width translates correctly. Just need to add `data-testid="stCheckbox"` to the outermost div.
```javascript
  it("has correct className and style", () => {
    const props = getProps()
    render(<Checkbox {...props} />)
    const checkboxElement = screen.getByTestId("stCheckbox")
    
    expect(checkboxElement).toHaveClass("row-widget")
    expect(checkboxElement).toHaveClass("stCheckbox")
    expect(checkboxElement).toHaveStyle(`width: ${props.width}px`)
  })
```

Reply:
Thanks, added the test :) I used chatGPT to convert the tests and it told me to leave out this test 😅: 

<img width="663" alt="image" src="https://github.com/streamlit/streamlit/assets/2852129/b786486d-d910-418f-b0c6-2abf771990c1">